presto-hive/src/main/java/com/facebook/presto/hive/metastore/thrift/ThriftMetastoreUtil.java

```diff
@@ -401,6 +401,17 @@ public static HiveColumnStatistics fromMetastoreApiColumnStatistics(ColumnStatis
         return Optional.of(LocalDate.ofEpochDay(date.getDaysSinceEpoch()));
     }
 
+    /**
+     * Impala use -1 to indicate no-sample nullsCount
```

Comment:
- nit: "use*s*"
- This is not really a javadoc comment (it does not describe what the method is generally doing). I would replace it with inline `// Impala uses -1 to ...` comment inside `fromMetastoreNullsCount` method body

what is "no-sample nullsCount"?